### PR TITLE
[Bugfix] Condiment Runtime

### PR DESCRIPTION
**Pull request by AwkwardDryad**
## About the PR
//There was a runtime error that would happen literally every time condiments were applied to food.

- Fixed the thing :P

## Why's this needed?

- Runtime errors bad
- Runtime errors make code maid anger
- RuntimeErrorsMustDie.

### CODE REVIEW

File: code/modules/food_and_drink/condiments.dm [hunk 1]
```diff
@@ -17,12 +17,14 @@
...
+			return
 
 		if (istype(target, /obj/item/reagent_containers/))
 			user.visible_message("<span style='color:blue'><b>[user]</b> crushes up \the [src] in \the [target].</span>",\
 			"<span style='color:blue'>You crush up \the [src] in \the [target].</span>")
 			src.reagents.trans_to(target, 100)
 			qdel (src)
+			return
 
 		else return
```
Comment (ZeWaka) [inline code/modules/food_and_drink/condiments.dm:29]: useless line

File: code/modules/food_and_drink/condiments.dm [hunk 2]
```diff
@@ -17,12 +17,14 @@
...
 			src.reagents.trans_to(target, 100)
 			qdel (src)
+			return
 
 		if (istype(target, /obj/item/reagent_containers/))
 			user.visible_message("<span style='color:blue'><b>[user]</b> crushes up \the [src] in \the [target].</span>",\
 			"<span style='color:blue'>You crush up \the [src] in \the [target].</span>")
 			src.reagents.trans_to(target, 100)
 			qdel (src)
+			return
```
Comment (ZeWaka) [inline code/modules/food_and_drink/condiments.dm:27]: useless line

File: code/modules/food_and_drink/condiments.dm (hunk 1 above)
Comment (AwkwardDryad) [reply code/modules/food_and_drink/condiments.dm:29]: true.

File: code/modules/food_and_drink/condiments.dm (hunk 2 above)
Comment (AwkwardDryad) [reply code/modules/food_and_drink/condiments.dm:27]: this actually isnt useless. the lack of return statement here was what was causing the runtime in the first place.

Comment (ZeWaka) [reply code/modules/food_and_drink/condiments.dm:27]: the first return in this proc is not useless, this one certainly is

Comment (AwkwardDryad) [reply code/modules/food_and_drink/condiments.dm:27]: Oh i misread the comment order.

**Pull request merged**